adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala

```diff
@@ -141,20 +145,12 @@ sealed trait AlignmentRecordRDD extends AvroReadGroupGenomicRDD[AlignmentRecord,
   private[rdd] def maybeSaveBam(args: ADAMSaveAnyArgs,
                                 isSorted: Boolean = false): Boolean = {
 
-    if (args.outputPath.endsWith(".sam")) {
-      log.info("Saving data in SAM format")
-      saveAsSam(
-        args.outputPath,
-        asSingleFile = args.asSingleFile,
-        isSorted = isSorted
-      )
-      true
-    } else if (args.outputPath.endsWith(".bam")) {
-      log.info("Saving data in BAM format")
+    if (args.outputPath.endsWith(".sam") ||
+      args.outputPath.endsWith(".bam") ||
+      args.outputPath.endsWith(".cram")) {
```

Comment:
Yeah, I think I did something similar for VCF in https://github.com/bigdatagenomics/adam/pull/1104, although a private method and not a utility class. I'll get that on a review pass.